ivy/functional/frontends/numpy/statistics/histograms.py

```diff
@@ -0,0 +1,17 @@
+import ivy
+from ivy.functional.frontends.numpy import to_ivy_arrays_and_back
+
+
+
+@to_ivy_arrays_and_back
+
+def bincount(x, /, weights=None, minlength=None):
```

Comment:
The function structure is not correct should be
`bincount(x, /, weights=None, minlength=0)`